codebuild_specs/wait_for_ids.json

```diff
@@ -1,22 +1,20 @@
 [
-  "l_S3server_api_8_function_8",
   "l_amplify_app",
```

Comment:
what are the changes in this file?

Reply:
It's generated from https://github.com/aws-amplify/amplify-cli/blob/9034f865172fbc754a37c00783490ea972e02410/scripts/split-e2e-tests-codebuild.ts#L367-L368

Thanks @sobolk for pointing that out.